crashlog/usercrash.c

```diff
@@ -116,7 +116,12 @@ static int priv_process_usercrash_event(struct watch_entry *entry, struct inotif
             do_log_copy(eventname, dir, get_current_time_short(1), APLOG_TYPE);
             break;
         case HPROF_TYPE:
-            remove(path);
+            if (remove(path) == -1) {
+                LOGE("Failed to remove path %s\n", path);
+                free(key);
+                free(dir);
+                return -1;
```

Comment:
If you return here, other code logic may not be executed like raise_event.

Reply:
Ok, just check return value and print some logs